android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -134,22 +139,22 @@ protected void onActivityResult(
     }
 
     /**
-     * This method is called if the JS part does not handle the physical back
-     * button press.
-     */
-    @Override
-    public void invokeDefaultOnBackPressed() {
-        super.onBackPressed();
-    }
-
-    /**
      * {@inheritDoc}
      */
     @Override
     public void onBackPressed() {
         if (!JitsiMeetView.onBackPressed()) {
-            // Invoke the default handler if it wasn't handled by React.
-            super.onBackPressed();
+            // JitsiMeetView didn't handle the invocation of the back button.
+            // Generally, an Activity extender would very likely want to invoke
+            // Activity#onBackPressed(). For the sake of consistency with
+            // JitsiMeetView and within the Jitsi Meet SDK for Android though,
+            // JitsiMeetActivity does what JitsiMeetView would've done if it
+            // were able to handle the invocation.
+            if (defaultBackButtonImpl == null) {
```

Comment:
In what case 'defaultBackButtonImpl' will be null ? Is it possible for the activity to receive onBackPressed without being resumed ?